data_collection/gazette/spiders/ba_rio_do_pires.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class BaRiodoPiresSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "2926905"
+    name = "ba_rio_do_pires"
```

Comment:
About the execution:

Total of gazettes: 1746
First date: 2004-01-16 
Last date: 2021-08-13 ✅ 
Average of gazettes per day (6419 days): 0.3
Average of gazettes by month (211 months): 8.3
Average of gazettes by year (17 years): 102.7

Found 30 errors in log:
`sqlite3.IntegrityError: UNIQUE constraint failed: gazettes.territory_id, gazettes.date, gazettes.file_checksum`